tests/php/unit/Configuration/configuration-test.php

```diff
@@ -6,24 +6,38 @@
 use Brain\Monkey\Filters;
 use Brain\Monkey\Functions;
 
+/**
+ * Class Configuration_Test
+ *
+ * @package Yoast\AcfAnalysis\Tests\Configuration
```

Comment:
From the draft [PSR-5](https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#description-11) and also in the [PHPDocumentor documentation](https://docs.phpdoc.org/references/phpdoc/tags/package.html):

> The @package tag can be used as a counterpart or supplement to Namespaces. Namespaces provide a functional subdivision of "Structural Elements" where the @package tag can provide a logical subdivision in which way the elements can be grouped with a different hierarchy.
> 
> If, across the board, both logical and functional subdivisions are equal it is NOT RECOMMENDED to use the @package tag, to prevent maintenance overhead.

In other words: the `@package` tag can be dropped here (and in a number of other places).